### ISSUE TITLE
Colors

### ISSUE THREAD

**Issue by bpbond** [FoRTExperiment/fortedata#80]
```r
> print(forte_colors())
[1] "#000000" "#009E73" "#0072B2" "#D55E00"

> forte_colors
function () 
{
    pal <- c("#000000", "#009E73", "#0072B2", "#D55E00")
    pal <- structure(pal, class = "palette", name = "forte")
}
```

Which colors correspond to which treatments? This really needs to return a named vector.

**Comment by atkinsjeff** [comment]
it is in order 0, 45, 65, 85. The original way(s) I programmed it keep f-ing up terribly and it did not respond like a palette() function

**Comment by bpbond** [comment]
It looks like this was addressed by @atkinsjeff in 319ca2c9e96161962853a48c496fb5c8bfea7e37.
